adoc/bp_chap_intro.adoc

```diff
@@ -192,17 +192,19 @@ The following tables provide you with an overview of ports which are used by {su
 
 [[tab.install.ports.server]]
 .Required Server Ports
-[cols="1,1", options="header"]
+[cols="1,1,1", options="header"]
 |===
-| Port | Description
-| 67   | Required when {susemgr} is configured as a DHCP server for systems requesting IP addresses.
-| 69   | Used when {susemgr} is configured as a PXE server and allows installation and re-installation of PXE-boot enabled systems.
-| 80   | Used to contact {scc}. All WebGUI, client, and proxy server requests travel via http or https.
-| 443  | All WebGUI, client, and proxy server requests via http or https and {susemgr} uses this port for {scc} inbound traffic.
-| 5222 | When you wish to push actions to clients this port is required by the [systemitem]``osad`` daemon running on client systems.
-| 5269 | Needed if you push actions to or via a {susemgrproxy}.
-| 4505 | Required by the Salt-master to accept communication requests via TCP from minions. Inbound.
-| 4506 | Required by the Salt-master to accept communication requests via TCP from minions. Inbound.
+| Port | Protocol | Description
+| 22   | TCP      | SSH
+| 67   | TCP/UDP  | DHCP
+| 69   | TCP/UDP  | TFTP, used to support PXE services
+| 80   | TCP      | HTTP, used in some bootstrap cases
+| 123  | UDP      | NTP time service
+| 443  | TCP      | HTTPS, used for {webui}, client, and proxy server traffic
+| 4505 | TCP      | Salt, used by the Salt-master to accept communication requests from minions
```

Comment:
Salt uses two ZeroMQ sockets between the master and the minion (or proxy). Each of those consume one TCP connection.

First is on port 4505, connection is initiated by the minion and it is always open. master will send commands to minions through this channel (called a publish/subscribe topology, as the minion subscribes to notifications from the master).

Second is on port 4506, connection is initiated by the minion and it is kept open only when needed. In particular, minions will open this port when they have to report results to the master (typically, but not necessarily, after a command that had been received on port 4505 has finished). This is called a request/response topology as the minion sends "requests" to the master.

https://docs.saltstack.com/en/getstarted/system/communication.html

I think the above is good information to have somewhere here, or linked from here.